main.nf

```diff
@@ -1157,6 +1161,32 @@ def checkParameterList(list, realList) {
   return list.every{ checkParameterExistence(it, realList) }
 }
 
+def checkParams(it) {
+ // Check if params is in this given list
+  return it in [
+  'callName',
+  'call-name',
```

Comment:
What’s the difference between these (`callName` and `call-name`)? Do we need both?